### DISCUSSION TITLE
No waybar upon boot after update to latest dotfiles.

### DISCUSSION THREAD

**Discussion by LBTRS1911** [mylinuxforwork/dotfiles#585] [Q&A] [answered]
Since updating to the latest dotfiles, waybar does not show up when hyprland launches. Everything works perfectly except no waybar. I go into the dotfiles setup, toggle the show waybar slider off and back on and then waybar appears and everything is great. Any idea how to get it so waybar is there after rebooting and logging in?

**Comment by mylinuxforwork** [comment, marked as answer]
Can you open the file https://github.com/mylinuxforwork/dotfiles/blob/main/share%2Fdotfiles%2F.config%2Fhypr%2Fscripts%2Fxdg.sh

Scroll down and replace the sleep command at the end with sleep 3 and try again.

**Reply by LBTRS1911** [reply]
Thank you Stephan, it seems to have worked.

# Run waybar
sleep $_sleep3
~/.config/waybar/launch.sh

**Reply by jasonbarrow** [reply]
Thanks Stephan.  I just wanted to report that I had the same issue in case you're considering impact for a future update.  Your suggested fix worked for me too.
